overlord/snapstate/handlers.go

```diff
@@ -704,6 +704,8 @@ func downloadSnapParams(st *state.State, t *state.Task) (*SnapSetup, StoreServic
 	return snapsup, sto, user, nil
 }
 
+var storeDownloadIcon = store.DownloadIcon
```

Comment:
let's not complicate the mocking story, the store is already accessed through an interface which is swapped out for a mock in tests, there's a `fakeStore` implemented in `overlord/snapstate/backend_test.go`